cmd/workspace/apps/apps.go

```diff
@@ -77,7 +78,12 @@ func newCreate() *cobra.Command {
 	// TODO: short flags
 	cmd.Flags().Var(&createJson, "json", `either inline JSON string or @path/to/file.json with request body`)
 
-	cmd.Flags().StringVar(&createReq.Description, "description", createReq.Description, `The description of the app.`)
+	// TODO: complex arg: active_deployment
+	// TODO: complex arg: app_status
+	// TODO: complex arg: compute_status
+	cmd.Flags().StringVar(&createReq.App.DefaultSourceCodePath, "default-source-code-path", createReq.App.DefaultSourceCodePath, `The default workspace file system path of the source code from which app deployment are created.`)
```

Comment:
We need to filter this flag out because it's readonly, for this we need OUTPUT_ONLY being exposed in genkit cc @hectorcast-db @renaudhartert-db